doc/stim.pyi

```diff
@@ -6026,9 +6109,9 @@ class TableauSimulator:
             >>> def brute_force_post_select(qubit, desired_result):
             ...     global s
             ...     while True:
```

Comment:
Add an example that uses the `copy_rng=True` argument. For example, just measure in the X basis in the main and the copy and assert they are the same.

Reply:
Done.